packages/pyroscope-flamegraph/src/Toolbar.tsx

```diff
@@ -14,11 +14,15 @@ import useResizeObserver from '@react-hook/resize-observer';
 import Button from '@webapp/ui/Button';
 // eslint-disable-next-line import/no-extraneous-dependencies
 import Select from '@webapp/ui/Select';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
why do we need this line?

Reply:
this is my fault actually, the eslint rule thinks `@webapp` is an external dependency

this would be solved after we move `ui` to its own package, which is a good idea bc right now we have a circular dependency: `@webapp` uses flamegraph `flamegraph` which uses `@webapp/ui`